cmd/cloudflared/common_service.go

```diff
@@ -24,7 +24,8 @@ func getServiceExtraArgsFromCliArgs(c *cli.Context, log *zerolog.Logger) ([]stri
 		// currently, we only support extra args for token
 		return buildArgsForToken(c, log)
 	} else {
-		// empty extra args
-		return make([]string, 0), nil
+		return []string{
```

Comment:
Instead of limiting this to only be able to run with tunnel parameters, could we improve this to allow any tunnel sub-commands to be passed in?
For example, if someone wanted to run `cloudflared access tcp` as a service, we should allow them to also setup using this command similar to the following:
```
cloudflared service install -- access tcp --hostname test.example.com --url localhost:8080
```

We would still like to allow a user to provide just their tunnel token, so we can still leave and try and parse that first to insert the `tunnel run --token <TOKEN>` command, but if that fails, we should just provide whatever the user provides as a subcommand to cloudflared.

I think that solves the problem you are trying to solve here while also expanding the functionality to be expressive for other folks that might try something similar. What do you think?